.travis.yml

```diff
@@ -6,9 +6,12 @@ install:
 
 script:
   - tox -e py27
+  - (cd core && tox -e py27)
   - tox -e py34
+  - (cd core && tox -e py34)
   - tox -e lint
   - tox -e cover
+  - (cd core && tox -e cover)
```

Comment:
This is horribly unwieldy, and will only get worse as we break out other packages.